client/components/grade-table.jsx

```diff
@@ -8,7 +8,7 @@ export default function GradeTable(props) {
   }
 
   return (
-    <table className="table table-striped table-bordered col-10 offset-1">
+    <table className="table table-striped table-bordered col-9 ">
```

Comment:
Same as I mentioned on the form, instead of making the table a `col-9`, add a `div.col-9` to App's render and slot your table in there. The table and form should not know that they belong to a Bootstrap row.